app/assets/javascripts/comments.coffee

```diff
@@ -1,3 +1,17 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+$ ->
+  $('.save_edit').click ->
+      post_id = $('#post_id').val()
+      comment_id = $(this).parent().find('#comment_id').val()
```

Comment:
u can extract this for separate variable like `comment_item = $(this).parent()` and then use it